tests/integration/state.go

```diff
@@ -175,6 +175,9 @@ type state struct {
 	// nodes.
 	docIDs [][]client.DocID
 
+	// Valid Cid string values by [UniqueCid] ID.
+	cids map[any]string
```

Comment:
thought: I did consider making this a `map[any]any` - allowing validators to independently manage/extend state the same was as we can with a `context.Context`, but decided it was overkill/too-early to do that.  

It would mean that implementations of `Validator` would be entirely self contained - changing them or adding new ones would not require changes to any other types.